ProcessMaker/Http/Controllers/Api/ScriptController.php

```diff
@@ -202,6 +204,15 @@ public function store(Request $request)
         $request->validate(Script::rules());
         $script = new Script();
         $script->fill($request->input());
+
+        $userToRunAs = User::find($script->run_as_user_id);
+        if ($userToRunAs->tokens()->get()->count() === 0 && !$userToRunAs->is_administrator) {
+            return response(
+                ['message' => "The user doesn't have an access token",
+                    'errors' => ['run_as_user_id' => ["The user doesn't have an access token"]]],
```

Comment:
I shouldn't be able to choose a user if they don't have a token. If no users exist, you shouldn't be able to create a script and the modal should change in the same way the modal for adding processes changes when there are not process categories.